moveit_simple_controller_manager/src/moveit_simple_controller_manager.cpp

```diff
@@ -155,10 +166,13 @@ class MoveItSimpleControllerManager : public moveit_controller_manager::MoveItCo
           controllers_.erase(name);
           continue;
         }
-        
+
         /* add list of joints, used by controller manager and moveit */
         for (int j = 0 ; j < controller_list[i]["joints"].size() ; ++j)
+        {
           controllers_[name]->addJoint(std::string(controller_list[i]["joints"][j]));
+          ROS_INFO_STREAM("\n"<<std::string(controller_list[i]["joints"][j])<<"\n");
```

Comment:
this is probably pretty spammy and should be removed or changed to a DEBUG_NAMED